kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/test/java/org/kie/server/services/taskassigning/planning/SolutionSynchronizerTest.java

```diff
@@ -29,6 +29,7 @@
 
 import org.apache.commons.lang3.tuple.Pair;
 import org.junit.Test;
+import org.kie.api.task.model.Status;
```

Comment:
I guess this line was introduced by mistake, since it's not used around the test

Reply:
Done